packages/cli/src/lib/versioning/packages.ts

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright 2020 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { runPlain } from '../../lib/run';
+import { paths } from '../../lib/paths';
+
+const PREFIX = '@backstage';
+
+const DEP_TYPES = [
```

Comment:
Repeated. But that's OK I guess

Reply:
Yeah attempted a refactor to split out the other usage of this too, but wasn't worth it